### PR TITLE
Adding logging configuration to the driver in the operator.

### PR DESCRIPTION
**Pull request by mew2057**
* Leverages `--minloglevel` of glog to set the logging level of the operator and driver registrar pods.
* Added  an olm 2.0.1 entry to reflect API changes.

### CODE REVIEW

**Comment by Jainbrt** [comment]
@mew2057 is this a part 2.0.1 release, if not then please make the changes for 2.1.0 ?

File: operator/deploy/olm-catalog/ibm-spectrum-scale-csi-operator/2.0.1/csiscaleoperators.csi.ibm.com.crd.yaml [hunk 1]
```diff
@@ -0,0 +1,198 @@
+---
```
Comment (deeghuge) [inline operator/deploy/olm-catalog/ibm-spectrum-scale-csi-operator/2.0.1/csiscaleoperators.csi.ibm.com.crd.yaml:1]: Do we need this file ? same file will also come when we merge v2.0.1 into dev ?

**Comment by Jainbrt** [comment]
@mew2057 do you need any help here from our side ?

**Comment by mew2057** [comment]
@Jainbrt No, this should be good at this point.

**Comment by mew2057** [comment]
We might need to add that as well.

**Comment by deeghuge** [comment]
Closing as we will looking into other possible option for log level management with go operator

**Pull request closed without merging**